src/components/pages/rules/flyouts/ruleEditor/ruleEditor.js

```diff
@@ -46,17 +48,36 @@ const operatorOptions = [
   { label: '<=', value: 'LessThanOrEqual' },
   { label: '=', value: 'Equals' }
 ];
+
+const actionOptions = [
+  { label: 'Send SMS', value: 'SMS' },
+  { label: 'Send email', value: 'Email' },
+  { label: 'No action', value: 'none' }
```

Comment:
Removed actionOptions entirely, current default is email so options will eventually be pulled from telemetry endpoint like with devicegroups. Set default newAction() type to "Email" since this value is never shown to the user on the actual UI.